tests/core/management/commands/test_reset_deepl_budget.py

```diff
@@ -0,0 +1,33 @@
+from datetime import datetime
+
+import pytest
+
+from integreat_cms.cms.models import Region
+
+from ..utils import get_command_output
+
+
+# pylint: disable=unused-argument
+@pytest.mark.order("last")
+@pytest.mark.django_db(transaction=True, serialized_rollback=True)
+def test_reset_deepl_budget(load_test_data_transactional):
+    """
+    Ensure that DeepL budget gets reset successfully
+    """
+
+    current_month = datetime.now().month - 1
+    target_region_id = (
+        Region.objects.filter(deepl_renewal_month=current_month).first().id
```

Comment:
```suggestion
    region = Region.objects.create(
        slug="deepl_test",
        deepl_renewal_month=current_month,
        deepl_midyear_start_month=current_month + 1,
        deepl_budget_used=42,
```
I'd just generate the required region here.